### PR TITLE
FCM Push provider guide with iOS app [Ticket DVRL-115]

### PR DESCRIPTION
**Pull request by iampearceman**
https://github.com/novuhq/docs/assets/63902456/b11aa848-19bf-490d-8fe8-8ed4e4da2a52

### CODE REVIEW

File: guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx [hunk 1]
```diff
@@ -0,0 +1,1310 @@
+---
+title: 'How to send PUSH notifications to iOS devices with Firebase Cloud Messaging (FCM) using Novu'
+description: 'Leverage the digest functionality to send email notifications'
```
Comment (sumitsaurabh927) [inline guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx:3]: Digest?

Comment (sumitsaurabh927) [reply guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx:3]: This needs to change @iampearceman

File: guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx [hunk 2]
```diff
@@ -0,0 +1,1310 @@
...
+icon: 'circle-2'
+---
+## **Prerequisites**
+
+To complete this tutorial, you will need the following:
+
+- [Apple Developer](https://developer.apple.com/) membership (to obtain the required permissions to send push notifications)
+- Laptop running macOS to develop an iOS project
```
Comment (sumitsaurabh927) [inline guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx:11]: A machine running MacOS to work on an iOS project

File: guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx [hunk 3]
```diff
@@ -0,0 +1,1310 @@
...
+- [Firebase](https://firebase.google.com/) account
+- [Novu](https://web.novu.co/) account
+- [Xcode](https://developer.apple.com/xcode/) installed on your Mac
```
Comment (sumitsaurabh927) [inline guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx:14]: installed on your machine

File: guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx [hunk 4]
```diff
@@ -0,0 +1,1310 @@
...
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/2.png"/> </Frame>
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/3.png"/> </Frame>
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/4.png"/> </Frame>
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/5.png"/> </Frame>
+
+Let's now build and run our ios application and see if everything is working correctly
+
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/6.png"/> </Frame>
+
+Go to the firebase documentation website go to docs right here and select cloud messaging we're going to be following this guide to set up
```
Comment (sumitsaurabh927) [inline guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx:34]: select 'Cloud Messaging'

Comment (sumitsaurabh927) [reply guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx:34]: Go to the Firebase docs, and select 'Cloud Messaging'. We're going to follow this guide to add cloud messaging to our app.

File: guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx [hunk 5]
```diff
@@ -0,0 +1,1310 @@
...
+cloud messaging inside of the app
+
+you can read more about how firebase
```
Comment (sumitsaurabh927) [inline guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx:37]: You can read more about how Firebase Cloud Messaging works in their [official docs](link)

File: guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx [hunk 6]
```diff
@@ -0,0 +1,1310 @@
...
+cloud messaging works by reading the
+documentation here
+
+<Frame><img src="/guides/FCM-iOS-Novu/Assets/7.png"/> </Frame>
+
+Go over here to ios and click on set up
```
Comment (sumitsaurabh927) [inline guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu.mdx:43]: ios -> iOS

**Review by sumitsaurabh927 — CHANGES_REQUESTED** [review]
There are a lot of grammatical errors. Please run it through grammarly once @iampearceman 

I'll re-review it afterwards

**Review by unicodeveloper — APPROVED** [review]

**Comment by unicodeveloper** [comment]
Hey @iampearceman What about the link to the accompanying repo/code for this guide?

**Comment by jainpawan21** [comment]
Amazing work emil 🚀

**Review by jainpawan21 — APPROVED** [review]

**Review by sumitsaurabh927 — APPROVED** [review]

**Pull request merged**